src/Caliburn.Micro.Platform/Platforms/Maui/FormsApplication.cs

```diff
@@ -174,5 +183,12 @@ protected virtual IEnumerable<object> GetAllInstances(Type service)
         protected virtual void BuildUp(object instance)
         {
         }
+
+        /// <summary>
+        /// Called by the bootstrapper's constructor at design time to start the framework.
```

Comment:
TODO: Fix comment within `<summary>`. Same for:

- [src/Caliburn.Micro.Platform/Platforms/Maui/iOS/CaliburnApplicationDelegate.cs](https://github.com/Caliburn-Micro/Caliburn.Micro/pull/961/files#diff-efad0bf8ed095fb863f77090ddd962a1cfdb01411cb6da1ff85becb3a1b25c94)
- [src/Caliburn.Micro.Platform/Platforms/net46/Bootstrapper.cs](https://github.com/Caliburn-Micro/Caliburn.Micro/pull/961/files#diff-512cc7546d559b0a79139588ed589dd96dd6bee80311d69e72d563f611be031e)
- [src/Caliburn.Micro.Platform/Platforms/uap/CaliburnApplication.cs](https://github.com/Caliburn-Micro/Caliburn.Micro/pull/961/files#diff-793c776fc4641bf87c878264bb2a0b96395d1eeea4e4509d9f6ccf6da18fc1d6)